tests/unittests/cmd/devel/test_hotplug_hook.py

```diff
@@ -122,6 +124,18 @@ def test_successful_remove(self, mocks):
         mocks.m_activator.bring_up_interface.assert_not_called()
         init._write_to_cache.assert_called_once_with()
 
+    @pytest.mark.parametrize("skip", [True, False])
+    def test_skip_detected(self, skip, mocks):
```

Comment:
A super nit comment (feel free to ignore):

If I understand correctly, this relies on the fact that `event_handler.detect_hotplugged_device` raises a `ValueError` if `udevaction` is unknown and that function is only called if `skip_hotplug_detect` is false.

While that works at the moment, I find it a bit too complex for the reader to realize and also a bit fragile, as independent changes in `event_handler.detect_hotplugged_device` could break this test, e.g:

- Changing the raised exception
- The inclusion of `asdf` as a supported action (I know, very unlikely...)

Why not just mocking `event_handler.detect_hotplugged_device` and assert it is called depending on `skip_hotplug_detect`? In that way, this test would be easier to understand and more resilient.